### PR TITLE
Русский

### PR DESCRIPTION
**Pull request by vintprox**
Just translated strings to Russian, as native speaker would understand all these.

### CODE REVIEW

**Comment by vintprox** [comment]
Partly solves #8.
Took 2 actual days to make this translation. By the way I wonder if anything in game is affected by localization strings?

What affects webAO, I plan to introduce substitute for

**Comment by stonedDiscord** [comment]
i changed some strings today, can you please update those

**Comment by vintprox** [comment]
I guess it should valid now.

**Comment by likeawindrammer** [comment]
I get `Unexpected characters 'history.replaceState({}, docum[...]' at AO2-Client/resource/translations/ao_ru.ts:219:5` and `Parse error at AO2-Client/resource/translations/ao_ru.ts:769:1: Extra content at end of document.`

File: resource/translations/ao_ru.ts [hunk 1]
```diff
@@ -0,0 +1,769 @@
...
+    </message>
+    <message>
+        <location filename="../../src/aooptionsdialog.cpp" line="197"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Enter as many callwords as you would like. These are case insensitive. Make sure to leave every callword in its own line!&lt;br&gt;Do not leave a line with a space at the end -- you will be alerted everytime someone uses a space in their messages.&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Введите на отдельных строках свои позывные, при указании которых в сообщениях будет подан звуковой сигнал.&lt;/body&gt;&lt;/html&gt;</translation>
+    </message>
+    <message>
+        <location filename="../../src/aooptionsdialog.cpp" line="203"/>
+        <source>Audio</source>
+        <translation>Аудио</translation>history.replaceState({}, document.title, clean_uri);
```
Comment (likeawindrammer) [inline resource/translations/ao_ru.ts]:
```suggestion
        <translation>Аудио</translation>
```

File: resource/translations/ao_ru.ts [hunk 2]
```diff
@@ -0,0 +1,769 @@
...
+        <translation>Ошибка</translation>
+    </message>
+    <message>
+        <location filename="../../src/debug_functions.cpp" line="23"/>
+        <source>Notice</source>
+        <translation>На заметку</translation>
+    </message>
+</context>
+</TS>
+;11M
```
Comment (likeawindrammer) [inline resource/translations/ao_ru.ts]:
```suggestion
```

**Comment by vintprox** [comment]
Ohh... that external keyboard disappoints me sometimes, including micro. Gotta remove these unexpected copypastas.

**Comment by vintprox** [comment]
Hm, I should have pressed "Commit suggestion"... instead I pushed changes myself.
Didn't know that desktop GitHub interface has such good tool as "suggestions".

**Pull request merged**